figures/scripts/fig1-sample-distribution.R

```diff
@@ -31,6 +31,8 @@ dir.create(supp_output_dir, recursive = TRUE, showWarnings = FALSE)
 
 # Metadata we'll use to calculate counts
 histologies_df <- read_tsv(file.path(data_dir, "pbta-histologies.tsv"))
+histologies_df <- histologies_df %>%
```

Comment:
Noting that we should add something into the 1B legend that unknown are excluded. I'll open that issue.

Reply:
this is a sneaky thing that comes and goes in new releases, since our clinical data is modified by sites all over the country at any given time 😅